colrev\_core.exceptions.OriginError
===================================

.. currentmodule:: colrev_core.exceptions

.. autoexception:: OriginError
